### DISCUSSION TITLE
Display beatmap title & artist during gameplay

### DISCUSSION THREAD

**Discussion by smileyhead** [ppy/osu#13128] [Ideas]
The main menu displays a song ticker in the top right corner whenever you switch songs. Could an option be added for that to be there permanently during gameplay as well?

**Comment by bdach** [comment]
You can't switch tracks in gameplay, so I'm confused as to what this would be doing. Do you mean just displaying the current beatmap title & artist?

**Comment by smileyhead** [comment]
Yeah.
